src/query-builder/QueryBuilder.ts

```diff
@@ -1405,7 +1405,7 @@ export class QueryBuilder<Entity> {
                 const updateSet = Object.keys(this.expressionMap.updateSet).map(key => key + "=:updateSet__" + key);
                 const params = Object.keys(this.expressionMap.updateSet).reduce((object, key) => {
                     // todo: map propertyNames to names ?
-                    object["updateSet_" + key] = this.expressionMap.updateSet![key];
+                    object["updateSet__" + key] = this.expressionMap.updateSet![key];
```

Comment:
when strict typescript compiler option enable it does not allow to access properties of objects without `if` check if it can be undefined. Here we explicitly tell compiler that `updateSet` cannot be undefined by using `!`